membership-attribute-service/app/actions/AuthAndBackendViaAuthLibAction.scala

```diff
@@ -2,7 +2,7 @@ package actions
 
 import com.gu.identity.auth.AccessScope
 import components.TouchpointBackends
-import filters.IsTestUser
+import filters.TestUserChecker
```

Comment:
I renamed this to make it less surprising, rather than having `IsTestUser` class with an `apply` method (as it sounds like a boolean)